pkg/algorithmregistry/algorithmregistry.go

```diff
@@ -33,6 +35,10 @@ var (
 		v1.PublicKeyDetails_PKIX_ED25519,
 		v1.PublicKeyDetails_PKIX_ED25519_PH,
 	}
+	// Opinionated ordering of hash algorithms from oldest and weakest to newest and strongest.
+	// Must be updated if AllowedClientSigningAlgorithms is updated with a new digest.
```

Comment:
I think we should add a comment above at AllowedClientSigningAlgorithms to update this.

Reply:
Added a comment.